src/features/surveys/repos/SurveysRepo.ts

```diff
@@ -233,6 +235,20 @@ export default class SurveysRepo {
     });
   }
 
+  getSurveys(orgId: number): IFuture<ZetkinSurveyExtended[]> {
+    const state = this._store.getState();
+    const surveyList = state.surveys.surveyList;
+
+    return loadListIfNecessary(surveyList, this._store, {
+      actionOnLoad: () => surveysLoad(),
+      actionOnSuccess: (data) => surveysLoaded(data),
+      loader: () =>
+        this._apiClient.get<ZetkinSurveyExtended[]>(
```

Comment:
This is not actually `ZetkinSurveyExtended[]`, but just `ZetkinSurvey[]` (without elements). See if changing the type here works. If not, save for a later refactor.

Reply:
in the `surveysLoaded` reducer on the store it takes the array and puts them in the `surveysList` in the store state, which is made up of `...Extended` objects, so I'm gonna make a note and leave it for later!